### PR TITLE
[Reviewer=RSL] Amend makefile to fix recursion error when running 'make *'

### PR DESCRIPTION
**Pull request by jack-atack**
Simple fix for issue #2, following a suggestion from dimbleby.

I've rebuilt the RPE (python process integrated with this library, which caught this issue), and functionality is unaffected

### CODE REVIEW

**Review by rowenasmithlamkin — COMMENTED** [review]
I think we should also update the line below to install distribute using pip as well.

However in the docs for virtualenv it has this:
"--distribute
--setuptools
Legacy; now have no effect. Before version 1.10 these could be used to choose whether to install Distribute or Setuptools into the created virtualenv. Distribute has now been merged into Setuptools, and the latter is always installed."

So assuming that the version of virtualenv is now later than 1.10  (I have 15.1 on my PC so that must be pretty old), I don't actually see why either of these libraries have to be installed separately! Maybe it's to make sure that setuptools is the correct version?
According to https://packaging.python.org/key_projects/#setuptools distribute was merged back into setuptools in version 0.7 so it's definitely not necessary!

In summary I think the changes should be:
   1. Remove the --setuptools flag from the virtualenv command.
   2. Change the setuptools step to use pip (as you have done)
   3. Get rid of the easy_install distribute step 
I've tried this on my VM and it all seems to work.

**Review by rowenasmithlamkin — APPROVED** [review]
Looks good to me
